contracts/cw721-base/src/msg.rs

```diff
@@ -157,6 +167,9 @@ pub enum QueryMsg<Q: JsonSchema> {
     /// Extension query
     #[returns(())]
     Extension { msg: Q },
+
+    #[returns(Option<String>)]
+    GetWithdrawAddress {},
```

Comment:
Prefer not having explicit getter in name per Rust convention, so just `WithdrawAddress {}`. Developer discretion.